doc/Core.xml

```diff
@@ -5352,6 +5362,592 @@ onvif://www.onvif.org/name/ARV-453
         </variablelist>
       </section>
     </section>
+
+  <section xml:id="s.snmpv3_extension">
+    <title>SNMPv3 Device Management Extension (Optional)</title>
+    <para>
+      This section defines an optional extension to the Device Management Service for supporting Simple Network Management Protocol version 3 (SNMPv3) as specified in <xref linkend="RFC3410"/> and related standards. Devices supporting this extension <emphasis role="bold">shall</emphasis> indicate this capability via the <literal>SNMPv3</literal> flag in the <literal>DeviceServiceCapabilities</literal> (see Section <xref linkend="_Ref468447902"/>).
```

Comment:
where SNMPv3 capability flag is added in DeviceServiceCapabilities in the PR?

Reply:
@ubkr @venki5685 there is an error in the table and the "SNMPv3" is not rendering properly in the table noted in my review comment. Looks to be missing a row section.